docs/data/sources/google-tag-manager-client.md

```diff
@@ -369,3 +339,33 @@ If you hardcode the value in your tag, the input will be forced into a string ty
 This video tutorial walks through the implementation basics. 
 
 <script src="https://fast.wistia.com/embed/medias/n337njhoot.jsonp" async></script><script src="https://fast.wistia.com/assets/external/E-v1.js" async></script><div class="wistia_responsive_padding" style="padding:56.25% 0 0 0;position:relative;"><div class="wistia_responsive_wrapper" style="height:100%;left:0;position:absolute;top:0;width:100%;"><div class="wistia_embed wistia_async_n337njhoot videoFoam=true" style="height:100%;position:relative;width:100%"><div class="wistia_swatch" style="height:100%;left:0;opacity:0;overflow:hidden;position:absolute;top:0;transition:opacity 200ms;width:100%;"><img src="https://fast.wistia.com/embed/medias/n337njhoot/swatch" style="filter:blur(5px);height:100%;object-fit:contain;width:100%;" alt="" aria-hidden="true" onload="this.parentNode.style.opacity=1;" /></div></div></div></div>
+
+## Breaking changes checklist
```

Comment:
I think we should link to this from the [warning above](https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/pull/904/files#diff-86171b1a11fcdb2c74f5676859fff852c835ef5693ca679ea5b507338d2a061bR12) to make it easy for folks to find.

Reply:
I will add this in other pr.